### ISSUE TITLE
Print the training set‘ evaluate result.

### ISSUE THREAD

**Issue by Echo9573** [sql-machine-learning/sqlflow#1351]
In the DNN model in tf.estimator, only the evaluation result of the validation set is output after the training is completed. It is recommended to print the training set to evaluate the effect after DNN model training.

**Comment by typhoonzero** [comment]
Maybe print both evaluation result on training set and validation set?

**Comment by Yancey0623** [comment]
Hi @typhoonzero , how about implementing a hook to static training metrics and print the training metrics at the end of each epoch?

**Comment by typhoonzero** [comment]
> Hi @typhoonzero , how about implementing a hook to static training metrics and print the training metrics at the end of each epoch?

Sure.

**Comment by typhoonzero** [comment]
After discuss with @Echo9573 , we are going to support: 

1. if verbose = 1, tf estimator models will output losses per several steps, Keras models will output losses per epoch. Metrics on training set and validation set will output when the training is finished.
1. if verbose = 0, only output metrics on training set and validation set when the training is finished.
